composer.json

```diff
@@ -9,13 +9,15 @@
         "php": "^7.1.3",
         "ext-ctype": "*",
         "ext-iconv": "*",
+        "ext-json": "*",
         "doctrine/doctrine-bundle": "^1.6",
         "doctrine/doctrine-fixtures-bundle": "^2.3",
         "doctrine/doctrine-migrations-bundle": "^1.2",
         "doctrine/orm": "^2.5",
         "friendsofsymfony/elastica-bundle": "^5.1.1",
         "friendsofsymfony/jsrouting-bundle": "^2.1",
         "friendsofsymfony/rest-bundle": "^2.5",
+        "gedmo/doctrine-extensions": "^2.4",
```

Comment:
This is an interesting library, but do we need it?

Reply:
Used to normalize file names, saw this on symfonycasts